app/AppKernel.php

```diff
@@ -35,12 +35,12 @@ public function getRootDir()
 
     public function getCacheDir()
     {
-        return dirname(__DIR__).'/var/cache/'.$this->getEnvironment();
```

Comment:
yeas, to avoid calling ``realpath``. Returning a non-normalized path can prevent Symfony from avoiding absolute paths in the cache files